content/en/blog/2022/experimental-dual-stack/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Support of Dual Stack Kubernetes Clusters"
+description: "Experimental and Alpha support for Dual Stack Kubernetes Clusters."
+publishdate: 2022-09-07T07:00:00-06:00
+attribution: "Steve Zhang (Intel), Alex Xu (Intel), Iris Ding (Intel), Jacob Delgado (F5), Ying-chun Cai (formerly F5)"
+keywords: [dual-stack]
+---
+
+Over the past year, both Intel and F5 have collaborated on an effort to bring support for
+[Kubernetes Dual-Stack networking](https://kubernetes.io/docs/concepts/services-networking/dual-stack/) to Istio.
+
+# Background
+
+The journey has taken us longer than anticipated and we continue to have work to do. Initially started with a design based
+on a reference design implementation from F5. The design lead to an [RFC](https://docs.google.com/document/d/1oT6pmRhOw7AtsldU0-HbfA0zA26j9LYiBD_eepeErsQ/edit?usp=sharing) that
+caused us to re-examine our approach. Notably, there were concerns about memory and performance issues that the community wanted
+to be addressed before implementation.
+
+## Experimental Dual Stack branch
+
+While the team was re-evaluating its options we created a new branch, [experimental-dual-stack](https://github.com/istio/istio/tree/experimental-dual-stack),
+to experiment and implement the reference design as we continued to learn. We will detail how to use and install artifacts
+from that branch in an upcoming blog, however, be aware that its initial intent was to learn how to approach development
+in a way that we can implement a big impacting feature within Istio without causing regressions. As such, it would be
+considered highly experimental to use--unit and integration tests do not currently pass--yet there are still people
+that are testing and using this in their environment.
+
+A big part of the original design was to duplicate a lot of the Envoy configuration. Notably, the listeners, clusters and routes
+with a new set of endpoints they can be referred to. The original design was created with customer requirements that
+specified client originating IPv4 requests should be proxied over IPv4, and the same for an originating IPv6 to be
+satisfied over IPv6 (we will refer to this as native IP family going forward).
+
+## Community Feedback
+
+Much of the feedback provided by the community for the original RFC was to change Envoy to better support dual-stack use cases
+internally instead of supporting this within Istio. This has led us to a [new design](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing)
+where we have taken lessons learned as well as feedback and have applied them.
+
+## Current Work
```

Comment:
New section

## Upcoming Istio 1.17

<wait for @zhlsunshine comments>

Reply:
1. Gateways work with dual-stack
2. In-mesh services now can be dual stack

Limitations
1. Connecting draining issues
2. Isn't fully native IPv4 downstream and IPv4 upstream (same for IPv6); many users don't care about this

Reply:
Is there anything you have done for ztunnel?